spinalcordtoolbox/deepseg/models.py

```diff
@@ -399,14 +399,23 @@ def install_model(name_model):
     # List of mirror URLs corresponding to a single model
     if isinstance(url_field, list):
         model_urls = url_field
-        download.install_data(model_urls, folder(name_model))
+        urls_used = [download.install_data(model_urls, folder(name_model))]
```

Comment:
I think this can be a string, rather than a list with a single string in it.
```suggestion
        urls_used = download.install_data(model_urls, folder(name_model))
```